testdata/README.md

```diff
@@ -0,0 +1,27 @@
+# librsync-go Test Data
+
+Reference files (`*.signature`, `*.delta`) were created using the original (C
```

Comment:
Is it worth it to have a script/function that could be used to generate the test data with `rdiff`? in the spirit of the stdlib's golden files https://github.com/golang/go/blob/master/src/cmd/internal/test2json/test2json_test.go#L21

I guess it's not necessary to be done again right?

Reply:
Hum, good question. We should not need to generate _these_ files again but someone probably will  want to add more test files in the future. I added one more commit explaining how to I ran `rdiff` to generate them.

Reply:
nice o/